### DISCUSSION TITLE
Bold 글씨 업데이트 시간 문의드립니다.

### DISCUSSION THREAD

**Discussion by adf2315** [kimytsc/covid-rest-vaccine-macro#52] [Q&A] [answered]
delay는 1000으로 설정했습니다.

이미지 내의 업데이트시간은 업데이트를 1초단위로 하고 있지만

 Bold 글씨는 4-5초 단위로 움직이는것 같습니다. 

어제 시도 했을때만해도 비슷하게 업데이트가 됐었는데 오늘 갑자기 왜이러는지 모르겠습니다.

그래도 업데이트는 꾸준히 되고 있다면 문제가 없는건가요??

**Comment by kimytsc** [comment, marked as answer]
회색으로 표시된 업데이트 시도 시간은 네이버 API를 호출한 시간으로 delay가 1000(1초)이므로 1초단위로 변경이 됩니다.

Bold로 표시되는 시간은 네이버의 DB가 갱신된 추정 시간으로, API에서 `vaccineLastSave`값으로 전달받은 시간을 그대로 노출하였습니다.

트래픽이 몰려 업데이트가 지연된다면, Bold로 표시되는 시간이 멈춰있을 것으로 판단되고, 4~5초 단위로 업데이트가 되고 있다면 정상적으로 업데이트 되고 있는 상황입니다.

**Reply by adf2315** [reply]
답변 감사합니다!! ㅎㅎ 잘 사용해서 예약성공하겠습니다. 감사합니다
